nuxt/pages/about.vue

```diff
@@ -18,27 +20,35 @@
           small
         >
           <template v-slot:opposite>
-            <span class="headline font-weight-bold" v-text="date"></span>
+            <span class="headline font-weight-bold" v-text="date" />
           </template>
           <div class="py-4">
-            <span v-if="$vuetify.breakpoint.mdAndDown"
-                  class="headline font-weight-bold"
-                  v-text="date"></span>
-            <h2 class="headline font-weight-light mb-4">{{ headline }}</h2>
-            <div v-html="text"></div>
+            <span
+              v-if="$vuetify.breakpoint.mdAndDown"
+              class="headline font-weight-bold"
+              v-text="date"
+            />
+            <h2 class="headline font-weight-light mb-4">
+              {{ headline }}
+            </h2>
+            <div v-html="text" />
           </div>
         </v-timeline-item>
       </v-timeline>
     </v-container>
     <v-container class="app__section">
-      <h5 class="section__title text-center mt-4 mb-8 display-1">Credits</h5>
+      <h5 class="section__title text-center mt-4 mb-8 display-1">
+        Credits
+      </h5>
       <v-row>
-        <v-col cols="12" lg="4" v-for="(contributor, index) in contributors" :key="index">
+        <v-col v-for="(contributor, index) in contributors" :key="index" cols="12" lg="4">
```

Comment:
Can we simplify these `index` variables? 

```suggestion
        <v-col v-for="(contributor, i) in contributors" :key="i" cols="12" lg="4">
```